src/Microsoft.DotNet.Darc/DarcLib/Helpers/ManifestHelper.cs

```diff
@@ -102,4 +113,73 @@ List<string> GetTargetPaths(DownloadedAsset asset)
         }
         return JObject.FromObject(manifestObject, JsonSerializer.CreateDefault(new JsonSerializerSettings() { NullValueHandling = NullValueHandling.Ignore }));
     }
+
+    private static List<DownloadedAsset> SelectMergedManifestAssets(IEnumerable<DownloadedBuild> downloadedBuilds)
+    {
+        return downloadedBuilds.Select(build =>
+            build.DownloadedAssets.Where(asset =>
+                asset.Asset.Name.EndsWith(MergedManifestFileName)))
+            .SelectMany(x => x).ToList();
+    }
+
+    private static Dictionary<string, string> RetrieveAssetOwnerMap(IEnumerable<DownloadedAsset> mergedManifests, ILogger logger)
+    {
+        Dictionary<string, string> assetOwnerMap = new();
+
+        foreach (var mergedManifest in mergedManifests)
+        {
+            XDocument document = null;
```

Comment:
Can we turn on nullability in this file?

Reply:
Good point. At one point, I think, we 'd need to turn on nullability for the whole project. It'd allow to detect some bugs that are present there.